spec/requests/workers_controller_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rails_helper'
+
+RSpec.describe "WorkersController", type: :request do
+  let(:sample_org_user) { create(:user) }
+  let(:sample_worker_user) { create(:user, :worker_user_type) }
+  # let(:sample_worker) { create(:worker, user_id: sample_worker_user.id) }
```

Comment:
Hey @ncbui please delete these 2 comments, then I'll merge

Reply:
Oops, thanks for catching this. Have removed.